src/http-manager.ts

```diff
@@ -5,16 +5,25 @@ type ServiceIds = {
 };
 
 /**
- * Assign numeric ids to every http requests and the corresponding responses
+ * Assigns numeric IDs to every http request and the corresponding response
  * through {@link AnswersCore}. This helps track the received order of requests
- * and responses. {@link AnswersHeadless} use it to ensure dispatch event for
- * state update is trigger from up-to-date responses (e.g. if the new received
- * response have higher id number from the recorded received response)
+ * and responses. {@link AnswersHeadless} uses it to ensure dispatch events for
+ * state updates are triggered with up-to-date responses (e.g. if the newly received
+ * response has a higher ID number than the recorded received response).
```

Comment:
I believe this class is internal

Reply:
Yes, I'll remove some docs I added for this class, `ReduxStateManager`, `HttpManager`, and others that are internal